src/main/webapp/app/modules/conflicts/multiple/multiple-record-view.tsx

```diff
@@ -69,7 +69,7 @@ export class MultipleRecordView extends React.Component<IMultipleRecordViewProp,
               <h4 className="from">
                 <Translate contentKey="multiRecordView.yourData" />
               </h4>
-              <Details organization={baseRecord.record.organization} {...this.props} exclusions={baseRecord.record.exclusions} />
+              <Details activity={baseRecord} {...this.props} exclusions={baseRecord.record.exclusions} isBaseRecord />
```

Comment:
Why 'isBaseRecord' default value is set to true? There will be always more children.

Reply:
It's for the multi-record-view to indicate how some elements should be visible